pkg/controller/podgc/gc_controller.go

```diff
@@ -44,7 +44,9 @@ const (
 
 type PodGCController struct {
 	kubeClient              clientset.Interface
-	resourceProviderClients map[string]clientset.Interface
+
+	// all clients to list nodes it cares about, particularly including the current TP client
+	nodeListers map[string]clientset.Interface
```

Comment:
We typically use resourceListers for special purpose. Such as podLister in lin 51. Based on kubeClient, perhaps this one can be renamed as kubeClientForNodes or resourceClients, etc.

Reply:
renamed as suggested.